.github/workflows/test.yml

```diff
@@ -0,0 +1,36 @@
+name: build
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build-test:
+    name: Build and Test
```

Comment:
Mabe we should add code styling with inspectcode?

Reply:
Probably. Should be a different workflow though.

Reply:
True.